src/Compilers/CSharp/Portable/Emitter/EditAndContinue/CSharpSymbolMatcher.cs

```diff
@@ -718,6 +728,9 @@ private bool AreNamedTypesEqual(NamedTypeSymbol type, NamedTypeSymbol other)
                 // TODO: Test with overloads (from PE base class?) that have modifiers.
                 Debug.Assert(!type.HasTypeArgumentsCustomModifiers);
                 Debug.Assert(!other.HasTypeArgumentsCustomModifiers);
+                Debug.Assert(!type.IsTupleType);
+                Debug.Assert(!other.IsTupleType);
```

Comment:
A type argument can be a tuple type. Or did you mean to say: "but that path only passes type **parameters** for comparison"?

---
In reply to: [74845218](https://github.com/dotnet/roslyn/pull/13024#discussion_r74845218) [](ancestors = 74845218)